bundler/doc/development/SETUP.md

```diff
@@ -12,18 +12,15 @@ To work on Bundler, you'll probably want to do a couple of things:
 
         $ brew install graphviz
 
-* Install development dependencies from the rubygems root directory:
+* Install development dependencies and Bundler's test dependencies from the rubygems root directory:
 
         $ rake setup
+        $ bin/rake spec:parallel_deps
```

Comment:
Instead of joining these, I could also apply the format from e.g. [CONTRIBUTING.md](https://github.com/rubygems/rubygems/blob/468e166dcdbcbc2ccf8bf2e93ed80c2f6e04af28/CONTRIBUTING.md) and remove the leading `$` so that GitHub's handy little copy-to-clipboard button works directly. 🤓

Reply:
This could also say `bin/rake setup spec:parallel_deps`.

Reply:
Haha, yes. 🤦🏼 

Changed it accordingly.